default.nix

```diff
@@ -106,7 +106,9 @@ in rec {
   pathGit = ./.;  # Used in CI by the migration graph hash algorithm to correctly ignore files.
   path = reflex-platform.filterGit ./.;
   obelisk = ghcObelisk;
-  obeliskEnvs = ghcObeliskEnvs;
+  obeliskEnvs = ghcObeliskEnvs // {
```

Comment:
Maybe we should just filter ghcObelisk for just the `obelisk-*` attrs? I'm not sure whether we have similar collisions like this.

Reply:
Good idea

Reply:
I did this instead.